API/projectClass/domainsClass.m

```diff
@@ -71,106 +113,389 @@
         end
     
         % ----------------------------------------------------------------
-        %
-        %   Editing of Domains Contrasts Block
+        % Editing of Domains Contrasts Block
+
+        function obj = addContrast(obj, options)
```

Comment:
Are these routines now required here because of the change to an argument block?
Also, the above should read "% Editing of Contrasts Block" - the domain contrasts come later.

Reply:
Please again refer to the previous comments.

Reply:
The argument blocks are different hence the duplication